tests/js/server/aql/aql-index-hints.js

```diff
@@ -451,7 +468,27 @@ function indexHintSuite() {
             assertNotEqual(indexHint[1].indexOf(node.indexes[0].name), -1);
         });
       });
-    }
+    },
+
+    testForceAnotherIndexWithGeoPresent: function () {
+      const query = `FOR d IN ${cn2}  OPTIONS {indexHint: "${invertedIdxName}", forceIndexHint: true, waitForSync: true} 
+      FILTER GEO_DISTANCE([50, 50], d.geo) < 5000 
+      RETURN d._key`;
+      const usedIndexes = getIndexNames(query);
+      assertEqual(usedIndexes.length, 1);
+      assertEqual(usedIndexes[0].length, 1);
+      assertEqual(usedIndexes[0][0], invertedIdxName);
+    },
+
+    testAnotherIndexHintNoForceWithGeoPresent: function () {
+      const query = `FOR d IN ${cn2}  OPTIONS {indexHint: "${invertedIdxName}", waitForSync: true} 
```

Comment:
same
```suggestion
      const query = `FOR d IN ${cn2}  OPTIONS {indexHint: "${invertedIdxName}"} 
```

Reply:
Now we need this as we actually run query